src/routes/usergroups.ts

```diff
@@ -2,4 +2,377 @@ import * as express from "express";
 import * as db from '../tools/db';
 
 export const register = (app: express.Application) => {
+    /* /usergroup endpoint */
+    
+    // Get method
+    // get a list of all usergroups
+    app.get('/api/v1/usergroup', (req: any, res) => {
```

Comment:
Don't use implicit `any` please. You should use this `(req: Request, res: Response)`